src/CattyTests/DataModel/SceneTest.swift

```diff
@@ -283,4 +283,18 @@ final class SceneTest: XCTestCase {
         XCTAssertTrue(scene.isEqual(copyScene))
         XCTAssertFalse(copyScene === scene)
     }
+
+    func testCopyObjects() {
+        let object1 = SpriteObject()
+        object1.name = "testObject1"
+        scene.add(object: object1)
+
+        let object2 = SpriteObject()
+        object2.name = "testObject2"
+        scene.add(object: object2)
+
+        let objectsBefore = scene.count
+        scene.copyObjects([object1, object2])
```

Comment:
This line is throwing a warning in Xcode. Please remove return value if unused or use return value in test (e.g., to check that the returned objects are also part of the objects). Casting the return value to `void` should not be a solution.